transform/instance.go

```diff
@@ -367,6 +367,76 @@ func (ot *objectTransformer) transform(in interface{}, modifier pathModifier) (i
 
 }
 
+// transform retrieves the value for this object by building the value for the base object and then adding in any
+// transforms for all defined child fields. If a transform is provided it transforms the children relative to the
+// passed in node
+func (ot *objectTransformer) objectTransformXML(in interface{}, modifier pathModifier) (interface{}, error) {
+	path := ot.jsonPath
+	if modifier != nil {
+		path = modifier(path)
+	}
+	var newValue map[string]interface{}
+
+	// For the object use a transform if it exists or the default or an empty map
+	if ot.transforms != nil {
+		rawValue, err := ot.transforms.transform(in, "object", modifier, ot.format)
+		if err != nil {
+			return nil, err
+		}
+		newValue = make(map[string]interface{})
```

Comment:
I'm not sure this needs to be defined here, the code in lines 400-406 seems to take care of it

Reply:
good call